client/state/happiness-engineers/reducer.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+import { keyBy } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { createReducer } from 'state/utils';
+import { itemsSchema } from './schema';
+import {
+	HAPPINESS_ENGINEERS_FETCH,
+	HAPPINESS_ENGINEERS_RECEIVE,
+	HAPPINESS_ENGINEERS_FETCH_FAILURE,
+	HAPPINESS_ENGINEERS_FETCH_SUCCESS
+} from 'state/action-types';
+
+/**
+ * Returns the updated requesting state after an action has been dispatched.
+ * Requesting state tracks whether a network request is in progress for a site.
+ *
+ * @param  {Object} state  Current state
+ * @param  {Object} action Action object
+ * @return {Object}        Updated state
+ */
+export const requesting = createReducer( false, {
+	[ HAPPINESS_ENGINEERS_FETCH ]: () => {
```

Comment:
This could be shortened to

```js
[ HAPPINESS_ENGINEERS_FETCH ]: () => true
```

which I find more elegant (same for the two actions below).